TLM/TLM/UI/TrafficManagerTool.cs

```diff
@@ -615,13 +615,15 @@ void DefaultRenderOverlay(RenderManager.CameraInfo cameraInfo)
         private static float CalculateNodeRadius(ushort nodeId) {
             float sumHalfWidth = 0;
             int count = 0;
-            Constants.ServiceFactory.NetService.IterateNodeSegments(
-                nodeId,
-                (ushort segmentId, ref NetSegment segment) => {
-                    sumHalfWidth += segment.Info.m_halfWidth;
+            ref NetNode node = ref nodeId.ToNode();
+            for (int i = 0; i < 8; ++i) {
```

Comment:
For all use of 8 there was a const somewhere in ColossalOrder code.

Reply:
I haven't seen such `const` value in the decompiled code 😕

Reply:
Ok looks like indeed, there isn't. There are other good constants though in NetManager code, but not relevant to this review.